### ISSUE TITLE
docs: Add PR test report template

### ISSUE THREAD

**Issue by gaurivn** [anitab-org/vms#1020]
## Description
As a contributor for testing,
I need a template to test pull requests,
so that I can conveniently document the changes after testing a pull request.

### Update [Required]
- [ ] Create TEST_PR_TEMPLATE.md on docs folder

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Comment by 19-pragya** [comment]
can i work on this issue

**Comment by vaibhavarora102** [comment]
@gaurivn i would like to work on this in open source hack
can you please assign
thanks!

**Comment by gaurivn** [comment]
@vaibhavarora102, open-source hack issues can only be assigned starting September 1st, 2020.

**Comment by Abjcodes** [comment]
I would like to work on this issue as a part of OSH. Can you assign me this issue? Thank You.

**Comment by priyanshuone6** [comment]
Hey can you assign this issue to me, I would like to work on it

**Comment by gaurivn** [comment]
Sure assigning you @priyanshuone6

**Comment by priyanshuone6** [comment]
Do I need to just create a blank file named TEST_PR_TEMPLATE.md on docs folder?

**Comment by HeroicHitesh** [comment]
I would like to work on this issue. Please assign me this issue when it's free.

**Comment by gaurivn** [comment]
> Is TEST_PR_TEMPLATE and pull request template different?

Yes, they are different, I suggest you take a look at the other repositories' documentation and see how it's done. It's the same template across all repositories.

**Comment by gaurivn** [comment]
> I would like to work on this issue. Please assign me this issue when it's free.

Hi @HeroicHitesh, since this issue is already assigned, I can't assign you for time being. In case this issue becomes available in the future, will assign you.

**Comment by priyanshuone6** [comment]
> > Is TEST_PR_TEMPLATE and pull request template different?
> 
> Yes @priyanshuone6, they are different, I suggest you take a look at the other repositories' documentation and see how it's done. It's the same template across all repositories.

Something like this?
https://github.com/anitab-org/mentorship-backend/blob/develop/docs/test-pr-guide.md

**Comment by gaurivn** [comment]
@priyanshuone6, how is the progress

**Comment by vaibhavarora102** [comment]
if it's free kindly give me a chance

**Comment by gaurivn** [comment]
> if it's free kindly give me a chance

You're not a first-timer on this repo anymore.

**Comment by priyanshuone6** [comment]
It's almost done, I will make a pull request by the end of the day.
